docs/src/components/pricing/PricingTable.tsx

```diff
@@ -168,13 +174,60 @@ export function PlanPrice(props: PlanPriceProps) {
       </React.Fragment>
     );
   }
-  // else Premium
-  const premiumMonthlyValue = annual ? 49 : 49 * 3;
-  const premiumAnnualValue = premiumMonthlyValue * 12;
 
-  const premiumDisplayedValue = monthlyDisplay ? premiumMonthlyValue : premiumAnnualValue;
+  if (plan === 'premium') {
+    const premiumMonthlyValue = annual ? 49 : 49 * 3;
+    const premiumAnnualValue = premiumMonthlyValue * 12;
+
+    const premiumDisplayedValue = monthlyDisplay ? premiumMonthlyValue : premiumAnnualValue;
+    const priceExplanation = getPriceExplanation(
+      monthlyDisplay ? premiumAnnualValue : premiumMonthlyValue,
+    );
+
+    return (
+      <React.Fragment>
+        <LicenseModelSwitch />
+        <Box sx={{ display: 'flex', alignItems: 'center', justifyContent: 'center', mt: 1, mb: 4 }}>
+          <Box sx={{ width: 10 }} />
+          <Typography
+            variant="h3"
+            component="div"
+            sx={{ fontWeight: 'bold', color: 'primary.main' }}
+          >
+            {formatCurrency(premiumDisplayedValue)}
+          </Typography>
+          <Box sx={{ width: 5 }} />
+          <Typography variant="body2" sx={{ color: 'text.secondary', mt: '3px' }}>
+            {priceUnit}
+          </Typography>
+        </Box>
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'center',
+            gap: 2,
+            mb: 2,
+            minHeight: planPriceMinHeight,
+          }}
+        >
+          {(annual || monthlyDisplay) && (
+            <Typography variant="body2" sx={{ color: 'text.secondary', textAlign: 'center' }}>
+              {priceExplanation}
+            </Typography>
+          )}
+        </Box>
+      </React.Fragment>
+    );
+  }
+
+  // else enterprise
+  const enterpriseMonthlyValue = annual ? 79 : 79 * 3;
```

Comment:
AFAIK, to remove, the point of the enterprise plan is to not have any price listed.